frontend/src/common/components/ModifyOwnerModal.tsx

```diff
@@ -52,12 +52,12 @@ const OwnerMutateForm = ({owner, closeModalAction}: IOwnerMutateForm) => {
         mode: "all",
         resolver: resolver,
     });
-    const identifierValue = ownerFormObject.watch("identifier");
+    const ownerFormObjectValues = ownerFormObject.watch();
```

Comment:
If you only need one value from the formObject, you can do `ownerFormObject.watch("identifier");` 😉

Reply:
Watching all the fields was actually needed for the hasFormChanged to update. I changed it to use hook form's isDirty boolean, so I could change it back to watch only one field.